backend/seed_data/test_cases.yaml

```
@@ -5,43 +5,85 @@ test_cases:
+        outputs: 
+          - "[1,None,2,None,None,None,3]"
+      - input: "[1,None,2,None,None,None,3]"
+        outputs: 
+          - "[1,2,None,3]"
+
+  - module_type: "BONUS_CHALLENGE"
+    module_title: "Convert Sorted Array to Binary Search Tree"
+    target_runtime: O_N
+    test_cases:
+      - input: "[-10,-3,0,5,9]"
+        outputs: 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)